cpp/tests/strings/find_tests.cpp

```diff
@@ -162,6 +177,21 @@ TEST_F(StringsFindTest, ZeroSizeStringsColumn)
   EXPECT_EQ(results->size(), 0);
 }
 
+TEST_F(StringsFindTest, EmptyTarget)
+{
+  cudf::test::strings_column_wrapper strings({"Héllo", "thesé", "", "lease", "tést strings", ""},
+                                             {1, 1, 0, 1, 1, 1});
+  auto strings_view = cudf::strings_column_view(strings);
+
+  cudf::test::fixed_width_column_wrapper<bool> expected({1, 1, 1, 1, 1, 1}, {1, 1, 0, 1, 1, 1});
+  auto results = cudf::strings::contains(strings_view, cudf::string_scalar(""));
```

Comment:
What is `cudf::string_scalar("")` testing here?

Reply:
That an empty target will always return true (except for null rows).
https://github.com/rapidsai/cudf/pull/6598/files#diff-bca967de431009dab141d6336648cd0903895d87e563cfb9a6503a9ed7379d78R113

Reply:
I'm a little confused. I thought we were testing the `f(column, column)` variant. does the scalar get converted to a column?

Reply:
Also, what happens if the target column contains nulls?

Reply:
> I'm a little confused. I thought we were testing the `f(column, column)` variant. does the scalar get converted to a column?

This code was not added/changed in this PR. I just moved the code around in the test file a bit as mentioned in the PR description.

> Also, what happens if the target column contains nulls?

Answered above in a previous comment.
Null `targets[i]` will set the `output[i]` to false (unless `strings[i]` is null too then `output[i]` = null)